src/cli.js

```diff
@@ -88,7 +96,7 @@ shouldRebuildPromise
     if (!x) process.exit(0);
 
     return installNodeHeaders(argv.v, null, null, argv.a)
-      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, null, argv.a))
+      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, argv.c, null, argv.a))
```

Comment:
That's fair, I'd added it there as it felt like it made the most sense, but I can move it to the end as well :+1: